packages/gatsby-link/index.d.ts

```diff
@@ -1,11 +1,21 @@
 import * as React from "react";
 
-export const push: (to: LocationDescriptor) => void;
-export const replace: (to: LocationDescriptor) => void;
+export const push: (to: string) => void;
+export const replace: (to: string) => void;
 
 // TODO: Remove navigateTo for Gatsby v3
-export const navigateTo: (to: LocationDescriptor) => void;
+export const navigateTo: (to: string) => void;
 
 export const withPrefix: (path: string) => string;
 
-export default class GatsbyLink extends React.Component<GatsbyLinkProps, any> { }
+export interface GatsbyLinkProps extends NavLinkProps {
+  onClick?: (event: any) => void;
+  innerRef?: (instance: any) => void;
+  className?: string;
+  activeClassName?: string;
+  style?: { [key: string]: any };
```

Comment:
This should be an object

Reply:
Why not `React.CSSProperties`?
That way you get code completion on the style prop! :clap:

Reply:
ooo nice idea!

Reply:
@porfirioribeiro That's great, thanks!

@KyleAMathews The first way defines an object with the key as a string and any type for the value. It would work but React.CSSProperties is certainly the better _route_ :)